cfx/README.md

```diff
@@ -34,14 +34,10 @@ To get this extension working for you and your domains, you'll need to fork it a
 `./lib/main.js`:
 
 ```js
-var allowedUris = [
-  'https://respoke.github.io/*',
-  '*.ngrok.com'
-];
-
 var allowedDomains = [
-  'respoke.github.io',
-  '*.ngrok.com'
```

Comment:
@chadxz I didn't have to increase the security during testing, but the tabs api was only used during debugging so I went ahead and removed it.